packages/angular/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "tsup";
+
+const supportedPackages = ['data-connect'];
+export default defineConfig({
+	entry: [`src/data-connect/index.ts`, 'src/index.ts'],
+	format: ["esm"],
+	dts: true, // generates .d.ts files
+	outDir: "./",
```

Comment:
how if a directory such as `dist` is set as output dir just as in [react/tsup.config.ts](https://github.com/invertase/tanstack-query-firebase/blob/main/packages/react/tsup.config.ts).

Noticed an [angular/.gitignore](https://github.com/maneesht/tanstack-query-firebase/blob/main/packages/angular/.gitignore) has been introduced and currently seems to explicitly list build outputs to ignore....but there are some spill overs that are not being ignored such as [angular/index.d.ts](https://github.com/maneesht/tanstack-query-firebase/tree/main/packages/angular/index.d.ts)

seems safer if we had build outputs in a dist dir and ignored it entirely. in that case, [tanstack-query-firebase/.gitignore](https://github.com/maneesht/tanstack-query-firebase/tree/main/.gitignore) would ignore it

Reply:
I did it this way so that the `node` moduleResolution is supported. Unfortunately, if I put the output in the `dist` folder, then the output will be as follows:
```
dist/data-connect/...
package.json
```
where since `node` module resolution doesn't support `exports`, it will resolve to the path relative to the package.json, so if the user is trying to import `@tanstack-query-firebase/angular/data-connect`, it will look for a folder relative to the package.json and look for `/data-connect` instead of `./dist/data-connect`

Reply:
oh okay then

Reply:
I wrote a script that copies the package json into the dist directory. @Ehesp FYI, we'll have to make sure that the package.json that you publish is the one inside the dist folder.